app/views/devise/registrations/new.html.haml

```diff
@@ -8,7 +8,7 @@
           %p.error-message= resource.errors.full_messages_for(:email).join(', ')
       .field
         - if @minimum_password_length
-          - placeholder_text = "Password (#{@minimum_password_length} characters minimum)"
+          - placeholder_text = 'Password (#{@minimum_password_length} characters minimum)'
```

Comment:
Does this work for string interpolation?

Reply:
I changed it in the edit_user branch, but it seems that I lost this change when I did the rebase!